UtopiaCity/Services/Airport/FlightService.cs

```diff
@@ -67,5 +67,14 @@ public void DeleteFlight(Flight currentFlight)
             _dbContext.Remove(currentFlight);
             _dbContext.SaveChanges();
         }
+
+        /// <summary>
+        ///  Set the unique Flight Number
+        /// </summary>
+        /// <returns>Random integer value</returns>
+        public int GetRandomFlightNumber()
```

Comment:
probably value is not unique enough, either use computed for number as well, or increase complexity (use guid, or self-made analog)

Reply:
ok will change